public void onException(Throwable e){
    Rlog.e(LOG_TAG, "Error in onException", e);
}